eo-maven-plugin/src/main/java/org/eolang/maven/objectionary/ObjsDefault.java

```diff
@@ -23,43 +23,144 @@
  */
 package org.eolang.maven.objectionary;
 
+import java.io.IOException;
+import java.nio.file.Path;
 import java.util.HashMap;
 import java.util.Map;
+import org.cactoos.Input;
+import org.cactoos.Scalar;
+import org.cactoos.iterable.Mapped;
+import org.cactoos.map.MapEntry;
+import org.cactoos.map.MapOf;
+import org.cactoos.scalar.Unchecked;
+import org.eolang.maven.hash.ChCached;
+import org.eolang.maven.hash.ChNarrow;
 import org.eolang.maven.hash.CommitHash;
 
 /**
  * Default objectionaries.
+ * The class is immutable, but NOT thread-safe.
  * @since 0.29.6
  */
 public final class ObjsDefault implements Objectionaries {
+
+    /**
+     * Cache path.
+     */
+    private final Unchecked<Path> cache;
+
+    /**
+     * Use cache.
+     */
+    private final Scalar<Boolean> usehash;
```

Comment:
@volodya-lombrozo use**cache**?

Reply:
@maxonfjvipon Ok, I took a look and it seem that if `usecach==true` we use `OyRemote` and `OyHome` otherwise. So I believe we can name it `remote`. In this case if `remote == true` we use `OyRemote` and if `remote == false` we use `OyHome` (Objectionary from filesystem).